test/extended/router/h2spec.go

```diff
@@ -66,7 +66,7 @@ var _ = g.Describe("[sig-network-edge][Conformance][Area:Networking][Feature:Rou
 	})
 
 	g.Describe("The HAProxy router", func() {
-		g.It("should pass the h2spec conformance tests [apigroup:config.openshift.io][apigroup:authorization.openshift.io][apigroup:user.openshift.io][apigroup:security.openshift.io][apigroup:template.openshift.io]", func() {
+		g.It("should pass the h2spec conformance tests [apigroup:config.openshift.io][apigroup:authorization.openshift.io][apigroup:user.openshift.io][apigroup:security.openshift.io]", func() {
```

Comment:
You're still using the `router-shard.yaml` fixture at line 444, so I think it's premature to remove "[apigroup:template.openshift.io]" in this commit, but it isn't too important if a subsequent commit removes the use of the fixture.